modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostgisSimplifiedGeometryPreparedTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2020, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ *
+ */
+
+package org.geotools.data.postgis;
+
+import org.geotools.data.postgis.ps.PostGISPSTestSetup;
+import org.geotools.jdbc.JDBCTestSetup;
+
+public class PostgisSimplifiedGeometryPreparedTest extends PostgisSimplifiedGeometryTest {
```

Comment:
Checked out and verified the tests actually fail without the fix... they do, good! (the point read test reads 0,0 instead of the expected point)